### DISCUSSION TITLE
Tree planter error

### DISCUSSION THREAD

**Discussion by FPYS** [UMEP-dev/UMEP#342] [Q&A]
Hi,

I've got the following error using tree planter : 

File "C:/Users/Florian/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\processing_umep\processor\treeplanter_algorithm.py", line 357, in processAlgorithm
treerasters, positions = TreePlanterPrepare.treeplanter(cropped_rasters, treedata, treerasters, tmrt_1d)
File "C:/Users/Florian/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\processing_umep\functions\TreePlanter\TreePlanter\TreePlanterPrepare.py", line 53, in treeplanter
ts_temp1[y1:y2,x1:x2] = treerasters.treeshade
ValueError: could not broadcast input array from shape (69,125) into shape (69,124)
[Etat initial.zip](https://github.com/UMEP-dev/UMEP/files/7521477/Etat.initial.zip)

I've attached my files.

Thanks :)

**Comment by biglimp** [comment]
@nilswallenberg

**Comment by nilswallenberg** [comment]
Hi FPYS,

What settings did you use? Time steps, tree size, etc. Maybe you can send a print screen of your settings?

Cheers,
Nils

**Reply by FPYS** [reply]
Hi,

These are my settings. I also tried with greedy algorithm, same results. 
![image](https://user-images.githubusercontent.com/15260644/141478910-76526598-373e-46fb-85a6-ee1f51865d26.png)

**Reply by nilswallenberg** [reply]
I have now updated the code. Hopefully it works! I think you might have to run with UTC 0. I would also suggest to change the pixel resolution of your raster data to maybe 1 m to speed up things. The current pixel resolution is 0.2.

Cheers,
Nils
